cms/jinja2/templates/base.html

```diff
@@ -8,72 +8,76 @@
 {% endset %}
 
 {# fmt:off #}
-{% set languages = languages | default([
-    {
-        "url": "#",
-        "isoCode": "en",
-        "text": "English",
-        "current": true
-    },
-    {
-        "url": "#",
-        "isoCode": "cy",
-        "text": "Cymraeg",
-        "current": false
-    }
-])
-%}
+{% set languages=translation_urls() %}
 
 {% if page %}
     {% set body_class = ["template-", page.get_verbose_name()| lower | replace(" ", "-")] | join %}
 {% else %}
     {% set body_class = '' %}
 {% endif %}
 
+{# Untranslated pages will have aliases to the original page #}
+{% if page.alias_of and request.LANGUAGE_CODE != page.alias_of.locale.language_code %}
+    {% set localised_version_not_available = true %}
+{% endif %}
+
 {%- set page_title -%}
     {%- with current_site=wagtail_site() -%}
         {% if current_site and page.pk == current_site.root_page.pk and current_site.site_name %}{{ current_site.site_name }} - {% endif %}{% if page.seo_title %}{{ page.seo_title }}{% else %}{{ page.display_title | default(page.title) }}{% endif %}{% if current_site and page.pk != current_site.root_page.pk and current_site.site_name %} - {{ current_site.site_name }}{% endif %}
     {%- endwith -%}
 {%- endset -%}
 
 {% set navigation_settings = settings.navigation.NavigationSettings %}
-{% set main_menu = main_menu | default(navigation_settings.main_menu) %}
+{% set main_menu = main_menu | default(navigation_settings.main_menu.localized) %}
 
-{% set footer_menu = footer_menu | default(navigation_settings.footer_menu) %}
+{% set footer_menu = footer_menu | default(navigation_settings.footer_menu.localized) %}
+
+{# We set variables for text so that it gets picked up by makemessages #}
```

Comment:
```suggestion
{#- We set variables for text so that it gets picked up by makemessages -#}
```

for some whitespace cleanup.

ditto for the following `set`s